pycaret/time_series/forecasting/oop.py

```diff
@@ -3857,23 +3857,26 @@ def _plot_model(
 
             elif system:
                 if display_format == "streamlit":
-                    st.write(fig)
+                    if not return_fig:
```

Comment:
Since you are adding this if statement (`if not return_fig`) to every if-else block under (`elif system`), would it make sense to combine it with `elif system`?

e.g. `elif system and not return_fig:`

Reply:
Done. Now ready for review @ngupta23